old_cookbook/.gitignore

```diff
@@ -0,0 +1,160 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Should we just remove `old_cookbook`?

Reply:
Not yet, there are various notebooks and apps in the old notebook that either didn't have requirements.txt files, etc. I don't want to remove them but at the same time I want to clear a space so partners can start contribution notebooks and the .mdx files